games/applications/run_qpe.py

```diff
@@ -0,0 +1,149 @@
+# ------------------------------------------------------------------------------
+# Part of Red Queen Project.  This file is distributed under the Apache License.
+# See accompanying file /LICENSE for details.
+# ------------------------------------------------------------------------------
+
+# My code was inspired from the paper cited above. It provided a lot of directionality for the
+# structure of the code, however nothing is directly copied from it.
+# https://arxiv.org/pdf/2110.03137.pdf
```

Comment:
Code looks good, great Job! 

Consider linking the git repo from this paper in addition to the pdf.